### ISSUE TITLE
Incompatible to OctoPrint or the underlying Python environment

### ISSUE THREAD

**Issue by WouterKr** [edekeijzer/OctoPrint-PSUControl-HomeAssistant#8]
Hi,

Hope this is going to work. Would be a great addition to octoprint.

After installing the plugin it will not activate because it has a flash symbol next to it saying:

`**Incompatible to OctoPrint or the underlying Python environment**`

Is is possible solve this? I would really like to try the plugin.
Running octoprint 1.6.1

Thanks in advance.

**Comment by edekeijzer** [comment]
This sounds like you are still running on Python 2.7, which isn't actively supported by Octoprint anymore. Did you install from the plugin menu or did you have to copy/paste the link to the repository to install it?

**Comment by WouterKr** [comment]
Yes, I installed it by copying the url......
I feel stupid......I checked the version of python which was 3 and assumed it was using it also......
Also thought it updated the use of python3 by one of the latest updates.

So, manually updated to python3 and installed the plugin by the plugin menu and after configuring the plugins all is working. Thanks for the reply and this great addition to the PSU Control plugin!

Regards,
Wouter
